vs_utils/features/tests/nnscore_test.py

```diff
@@ -11,7 +11,7 @@
 import os
 import numpy as np
 import unittest
-import itertools
+#import itertools
```

Comment:
There are still sporadic bugs in the featurization, and having itertools around helps (I'll take it out once the code is a bit more stable).